packages/react-storage/src/components/StorageBrowser/views/LocationActionView/UploadView/UploadView.tsx

```diff
@@ -73,6 +75,8 @@ export function UploadView({
     ? getActionCompleteMessage({
         counts: statusCounts,
       })
+    : invalidFiles && !isProcessing
+    ? getFilesValidationMessage({ invalidFiles })
```

Comment:
Nit: now we are expanding to nested ternary expressions it could be extracted into a util function with clearer if flows. :D

Reply:
Good catch... it's a long day for me😵‍💫